sys/include/net/ndp.h

```diff
@@ -193,6 +193,18 @@ extern "C" {
 #define NDP_MAX_RANDOM_FACTOR       (1500U)     /**< MAX_RANDOM_FACTOR (x 1000) */
 /** @} */
 
+/**
+ * @brief   Hop-limit required for most NDP messages to ensure link-local
```

Comment:
Why `most` ? It seems that all message types uses this value, no ?

Reply:
I'm not so sure about that. Sure, all messages in RFC 4861 use it, but e.g. [DARs and DACs don't use it](https://tools.ietf.org/html/rfc6775#section-4.4).